apps/docs/docs/reference/sdk/faststore-sdk.md

```diff
@@ -1,38 +1,40 @@
 # FastStore SDK
 
-:::caution WIP
-This documentation is currently under development.
-:::
+FastStore SDK provides tools that help you deal with key ecommerce features:
 
-FastStore SDK is a simple, framework-agnostic implementation of Commerce APIs to help you create your next React-based store with world-class performance in record time.
+- [Shopping cart](#shopping-cart)
+- [Session](#session)
+- [Search](#search)
+- [Analytics](#analytics)
 
-## Key features
+:::info
+Note that the SDK manages these aspects of your store in the context of shoppers' browsers. This means, for example, that functions related to cart and session, do not necessarily send requests to the [FastStore API](/reference/api/faststore-api). In this case, the goal is to manage information in the browser, that can then be validated in the platform with the appropriate [mutations](/reference/api/mutations).
```

Comment:
```suggestion
Note that because these aspects are managed within the browser context, SDK functions do not necessarily send requests to the [FastStore API](/reference/api/faststore-api). In this case, to send information from the browser to the VTEX platform, it's necessary to use the appropriate mutations(/reference/api/mutations) from the FastStore API.
```